clicmd

```diff
@@ -58,6 +58,10 @@ function group_to_role {
   esac
 }
 
+if [[ -z ${GROUP} ]]; then
```

Comment:
It seems to me that the more correct way to solve the issue is fixing the line 50 to something like:
```
case "${1:-}"
```
and the error message at the line 55 to something like:
```
echo "Unable to determine the user's role"
```

What do you think?

Reply:
good idea for 'case "${1:-}"', but IMHO we still should test for -z explicitly.

Reply:
What for? 
I mean what is the difference between wrong non empty and just empty values in this case?

Reply:
Just to keep code clear to read